ttnn/cpp/ttnn/operations/data_movement/bcast/bcast.hpp

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#include "device/bcast_device_operation.hpp"
```

Comment:
Should avoid device operation include here. It helps to keep this include as a detail of implementation in cpp